types/sql.d.ts

```diff
@@ -1,7 +1,40 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License.
 
-import { FunctionInput, FunctionOutput } from './index';
+import { FunctionInput, FunctionOptions, FunctionOutput, FunctionResult, FunctionTrigger } from './index';
+import { InvocationContext } from './InvocationContext';
+
+export type SqlHandler = (changes: SqlChange[], context: InvocationContext) => FunctionResult;
+
+export interface SqlFunctionOptions extends SqlTriggerOptions, Partial<FunctionOptions> {
+    handler: SqlHandler;
+
+    trigger?: SqlTrigger;
+}
+
+export interface SqlTriggerOptions {
+    /**
+     * The name of the table monitored by the trigger.
+     */
+    tableName: string;
+
+    /**
+     * An app setting (or environment variable) with the connection string for the database containing the table monitored for changes
```

Comment:
For consistency, should we just use the description from our docs:

```
The name of an app setting that contains the connection string for the database against which the query or stored procedure is being executed.
```

Reply:
The existing comment is consistent with our other binding descriptions in this repo (cosmos, service bus, etc.), and I'd rather maintain consistency within the repo